app/api/helpers/scheduled_jobs.py

```diff
@@ -149,3 +152,18 @@ def expire_pending_tickets_after_three_days():
                                        (Order.created_at + datetime.timedelta(days=3)) <= datetime.datetime.now()).\
                                        update({'status': 'expired'})
         db.session.commit()
+
+
+def update_popular_event_locations():
+    from app import current_app as app
+    with app.app_context():
+        popular_locations = db.session.query(Event.searchable_location_name, func.count(Event.id).label('counts'))\
+            .group_by(Event.searchable_location_name)\
+            .order_by(desc('counts'))\
+            .limit(5)
+        db.session.query(EventLocation).delete()
```

Comment:
umm.. because if we only keep adding locations and not removing them we will have multiple copies of the same location. Every week say delhi will keep getting added. @mrsaicharan1